bin/install

```diff
@@ -34,7 +34,7 @@ install_poetry() {
   semver_ge "$ASDF_INSTALL_VERSION" 1.2.0 && vercomp="ge" || vercomp="lt"
 
   if [ $vercomp == "ge" ]; then
-    install_url="https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py"
+    install_url="https://install.python-poetry.org/install-poetry.py"
```

Comment:
```suggestion
    install_url="https://install.python-poetry.org"
```
I think this is probably safer given the poetry readme.

Reply:
Updated via rebase at 8b92b4f